.github/workflows/previewDeploy.yml

```diff
@@ -0,0 +1,52 @@
+name: Deploy to Cloudflare Pages
+
+on:
+  pull_request:
+
+jobs:
```

Comment:
needs a check to make sure it's a pr to `commaai/connect`. this will fail on PRs to forks